### PR TITLE
Changes to constants and such based on testing

### PR DESCRIPTION
**Pull request by JamisOrr**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -188,19 +188,22 @@ private void configureButtonBindings() {
...
-		m_operatorController.button(Button.kTriangle)
+		m_driverController.button(Button.kTriangle)
 				.onTrue(new ClimberPresetCommand(m_climberSubsystem, ClimberOperation.TOP,
 						() -> m_operatorController.getRawAxis(Axis.kLeftY),
 						() -> m_operatorController.getRawAxis(Axis.kRightY)));
-		m_operatorController.button(Button.kX)
+		m_driverController.button(Button.kX)
 				.onTrue(new ClimberPresetCommand(m_climberSubsystem, ClimberOperation.ZERO,
 						() -> m_operatorController.getRawAxis(Axis.kLeftY),
 						() -> m_operatorController.getRawAxis(Axis.kRightY)));
```
Comment (Gold856) [inline src/main/java/frc/robot/RobotContainer.java:198]: Should be all driver controller?

File: src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java [hunk 2]
```diff
@@ -47,6 +48,9 @@ public void initialize() {
 			case SET_PRESET_DEFAULT:
 				m_aimerSubsystem.setAimerHeight(kDefaultActuatorHeight);
 				break;
+			case SET_LOW:
+				m_aimerSubsystem.setAimerHeight(0.2);
```
Comment (Gold856) [inline src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java:52]: Make it a constant?

**Review by Gold856 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/RobotContainer.java:198]: No, this is correct

File: src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java (hunk 2 above)
Comment (JamisOrr) [reply src/main/java/frc/robot/commands/aimshooter/AimHeightCommand.java:52]: This position doesn't really matter, it's more of a testing thing, I can move it out if you want but like

**Review by Gold856 — APPROVED** [review]

**Pull request merged**